medic/medic-log.js

```diff
@@ -69,18 +70,72 @@ function logBlackberry() {
     return;
 }
 
-function logIOS() {
-    var logScriptpath = path.join("mobilespec", "platforms", "ios", "cordova", "console.log");
-    var command = "cat " + logScriptpath;
+function logIOS(appPath) {
+    // We need to print out the system log for the simulator app. In order to figure
+    // out the path to that file, we need to find the ID of the simulator running
+    // mobilespec
+
+    // First, figure out the simulator that ran mobilespec. "cordova run"" just chooses
+    // the last simulator in this list that starts with the word "iPhone"
+    shelljs.pushd(appPath);
+
+    var findSimCommand = getLocalCLI() + " run --list --emulator | grep ^iPhone | tail -n1";
```

Comment:
The `grep` and `tail` calls won't work on the Windows build machines. Is it a long task to do this in JS? If it's much more difficult, we can just add some steps to install Unix tools on Windows for our build machine docs.